ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleCCDClaimSubmissionOperationIndicators.java

```diff
@@ -21,18 +19,15 @@
             .claimIssueReceiptUpload(Y)
             .build();
 
-    Supplier<CCDClaimSubmissionOperationIndicators> getDefaultCCDClaimSubmissionOperationIndicators =
-        () -> getClaimSubmissionIndicatorWithVaue.get(CCDYesNoOption.NO, CCDYesNoOption.NO,
+    CCDClaimSubmissionOperationIndicators defaultCCDClaimSubmissionOperationIndicators =
+        getClaimSubmissionIndicatorWithVaue.get(CCDYesNoOption.NO, CCDYesNoOption.NO,
             CCDYesNoOption.NO, CCDYesNoOption.NO, CCDYesNoOption.NO, CCDYesNoOption.NO,
             CCDYesNoOption.NO);
 
-    Supplier<CCDClaimSubmissionOperationIndicators> getCCDClaimSubmissionOperationIndicatorsWithPinSuccess =
-        () -> getClaimSubmissionIndicatorWithVaue.get(CCDYesNoOption.NO, CCDYesNoOption.YES,
+    CCDClaimSubmissionOperationIndicators CCDClaimSubmissionOperationIndicatorsWithPinSuccess =
+        getClaimSubmissionIndicatorWithVaue.get(CCDYesNoOption.NO, CCDYesNoOption.YES,
```

Comment:
Declarations with Caps. Change it to init caps pls. Thanks

Reply:
👍